index.html

```diff
@@ -37,6 +46,12 @@
         </div>
     </nav>
 
+    <div class="container">
+        <div class="row">
+            <div id="tokens" class="col s12 center-align"></div>
+        </div>
+    </div>
+
```

Comment:
I don't have my laptop so i didn't download changes and view them in a browser, but is this your intention to have a tokens container as well as a View Tokens dropdown in the navbar?

Reply:
I added the id because thats how I'm targeting that html. the tokens container displays the tokens in the main page so the therapist doesnt have to click around between pages to use it